src/client/client.ts

```diff
@@ -164,6 +165,44 @@ export class TravelTimeClient {
     return this.request('/time-map/fast', 'post', { body, config: { headers } });
   }
 
+  async timeMapBatch(
+    bodies: TimeMapRequest[],
+    chunkSize?: number,
+  ): Promise<BatchedResponse<TimeMapResponse>>
+  async timeMapBatch<T extends keyof TimeMapResponseType>(
+    bodies: TimeMapRequest[],
+    format?: T,
+    chunkSize?: number,
+  ): Promise<BatchedResponse<TimeMapResponseType[T]>>
+  async timeMapBatch<T extends keyof TimeMapResponseType>(
+    bodies: TimeMapRequest[],
+    format: T,
+    chunkSize?: number,
+  ): Promise<BatchedResponse<TimeMapResponseType[T]>> {
+    const responses: TimeMapResponseType[T][] = [];
+    const errors: Array<{ index: number; error: Error }> = [];
+
+    for (let i = 0; i < bodies.length; i += chunkSize || 10) {
+      const chunk = bodies.slice(i, i + (chunkSize || 10));
```

Comment:
how would you slice intersections and unions here?

Reply:
discussed on call, no changes needed